src/sqltest/parts/insights/insightsDialogController.test.ts

```diff
@@ -50,6 +50,7 @@ suite('Insights Dialog Controller Tests', () => {
 			instMoq.object,
 			connMoq.object,
 			undefined,
+			undefined,
```

Comment:
These probably shouldn't be undefined - previously they'd just use cosole.log to log any messages (because they were using our custom log/warn/error functions). But now they'll fail if they ever hit a log statement.

So the question then is - do we just keep existing behavior and use TestLogService itself (which just no-ops the calls) or do we consider certain types of logging to be an error and fail the test (such as we're doing in the propertiesWidget test). I'm kind of in favor of doing the latter - while it is true that currently we don't currently consider error logs to be a test failure I'm of the opinion that they either should be failures or shouldn't be logging as an error (that should really only be used for critical issues IMO).

Reply:
I don't think logging should be the way to tell if a test has failed. there should be a more functional way to tell if a test failed.

Reply:
I definitely agree but I also wouldn't expect any console.error's to be logged during a passing test. But that can be a separate discussion - for now I'm fine as long as we keep existing behavior. 

We still shouldn't be using undefined though since as mentioned it'll cause a runtime exception if the component tries to log something. 

I'm not a fan of just swallowing the log messages regardless though - we should at the very least have the test output those messages in case there's something there to help debug the test.

Reply:
If you have a undefined reference, unless you are wrapping the test in weird ways with promises, it should fail the test.

Reply:
My point was that before it wouldn't fail the test (because we just logged to console.*), but now it would. And that would be the case for any logging statement, even log or warn (which could potentially be something that happens during a normal successful test run).

If you've run these tests and there haven't been any issues then I'm fine with it as is but for making the tests more robust I think it'd be worth it to just pass in the test logging service.